### PR TITLE
feat: change repo structure and package name

### PR DESCRIPTION
**Pull request by k-g-j**
## Description

Changes repository structure by removing packages directory and moving snap files into the root. Changes name of snap package from "@metamask/snap-watch-only-snap" to "@metamask/snap-watch-only"

Closes: https://github.com/MetaMask/accounts-planning/issues/382

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [1.0.1]
+
+### Changed
+
+- Converted to monorepo and changed package name to "@metamask/snap-watch-only"
```
Comment (gantunesr) [inline CHANGELOG.md:14]: Since this is not released yet you can remove this

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Account Watcher Snap
 
-This repository contains a simple example of a watch-only keyring snap.
+This repository contains a keyring snap for watch-only accounts.
```
Comment (gantunesr) [inline README.md:3]:
```suggestion
This repository contains a snap to manage watch-only accounts.
```

File: package.json [hunk 3]
```diff
@@ -1,8 +1,7 @@
 {
   "name": "@metamask/snap-watch-only",
-  "version": "1.0.0",
-  "private": true,
-  "description": "Monitor public addresses or ENS domains directly in MetaMask with this Watch-Only Snap.",
+  "version": "1.0.1",
+  "description": "Keep an eye on Ethereum addresses or ENS domains right in MetaMask",
```
Comment (gantunesr) [inline package.json:4]:
```suggestion
  "description": "Keep an eye on Ethereum accounts right in MetaMask",
```

File: snap.manifest.json [hunk 4]
```diff
@@ -1,18 +1,18 @@
 {
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (gantunesr) [inline snap.manifest.json:2]: The version we have today in NPM is 1.0.0, we should not change that unless we're doing a new release

Comment (k-g-j) [reply snap.manifest.json]: [e932ac9](https://github.com/MetaMask/snap-watch-only/pull/12/commits/e932ac956f923ea99b1ae9b18df00b7915c80b20)

File: CHANGELOG.md (hunk 1 above)
Comment (k-g-j) [reply CHANGELOG.md]: [e932ac9](https://github.com/MetaMask/snap-watch-only/pull/12/commits/e932ac956f923ea99b1ae9b18df00b7915c80b20)

**Review by gantunesr — APPROVED** [review]

**Pull request merged**